### PR TITLE
Logbook: "add logbook" form, logbook detail, and table of logbooks 

### PR DESCRIPTION
**Pull request by adrianpdm**
**task:**
1. insert CTA for contacting admin's Whatsapp if user's project is not included in list of available projects
2. remove startTaskTime and endTaskTime from logbook form
3. add view for logbook detail

**[global] FormInput--Input :**
- set `disabled` as props

**[global] FormInput--Select :**
- use `input` element instead of `select` when `disabled` prop is set to `true`

**[global] VeeValidate :**
- add `date_compare` rule that supports `gt, gte, lt, lte` operator for two date inputs comparison

### CODE REVIEW

**Comment by adrianpdm** [comment]
@yohang88 @Arif9878 @aldiiii

File: src/components/AttendanceButton.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        return this.checkout()
+      }
+    },
+    checkin () {
+      return this.$router.push({
+        path: '/checkins/create'
+      })
+    },
+    async checkout () {
+      await GroupwareAPI.post('attendance/checkout', {
```
Comment (aldiiii) [inline src/components/AttendanceButton.vue:54]: api sebaik nya setiap call API dibuat di store menjadi event, supaya seragam dengan yg lain ? 
cc: @adrianpdm

Comment (adrianpdm) [reply src/components/AttendanceButton.vue]: hatur nuhun,
sudah diperbaiki ya mang

**Comment by aldiiii** [comment]
build failed @adrianpdm error di `src/components/LogbookList/index.js`

File: src/components/AttendanceButton.vue (hunk 1 above)
Comment (aldiiii) [reply src/components/AttendanceButton.vue]: mantap mang.
Solved

**Comment by adrianpdm** [comment]
> build failed @adrianpdm error di `src/components/LogbookList/index.js`

sips, sudah diperbaiki ya

tadi build error saat linting, akibat indentasi `switch-case`

**Pull request merged**